### PR TITLE
docs: enable ga and cookie banner

### PR DESCRIPTION
**Pull request by m1ghtym0**
### Proposed change(s)

- Adds the google analytics plugin
- Enables the cookiebot banner

### Checklist

- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)

### CODE REVIEW

**Comment by m1ghtym0** [comment]
For testing: https://deploy-preview-86--edgeless-docs.netlify.app/constellation

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -177,6 +187,27 @@ async function createConfig() {
...
+                  label: 'Privacy Policy',
+                  href: 'https://www.edgeless.systems/privacy',
+                },
+                {
+                  html: `<a href="javascript: Cookiebot.renew()" class="footer__link-item">Cookie Settings</a>`
+                },
+
+                {
+                  label: 'Contact Us',
+                  href: 'https://www.edgeless.systems/contact/',
```
Comment (thomasten) [inline docs/docusaurus.config.js:207]: Is there any rule when you use `to` or `href`? If not, you can choose one of these:
* use `to` only for pages on docs.edgeless.systems, `href` for anything else
* use `to` for all pages on edgeless.systems, `href` for all external

The only difference seems to be that the href links get that arrow icon appended.
You'd also need to adapt some of the existing links.

Comment (m1ghtym0) [reply docs/docusaurus.config.js]: Yes, thanks for noticing. the first one was intended.

**Review by thomasten — APPROVED** [review]
can you change `to` to `href` for the Blog link, too?

**Pull request merged**